ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -1,54 +1,82 @@
 @page
 @model ntbs_service.Pages_Search.IndexModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
 @using Microsoft.AspNetCore.Routing
 
 @{
     ViewData["Title"] = "Search";
+    Layout = "../Shared/_NoForm_Layout.cshtml";
 }
 
 <h1>Search</h1>
-
-<h4>Parameters</h4>
-<hr />
 <div>
-    <form method="post">
+    <form method="get">
         <div asp-validation-summary="ModelOnly"></div>
-        <div>
+        <div class="enter-search-details-tip">
+            To search enter at least one of the following fields
         </div>
-        <div>
-            <input type="submit" value="Search" />
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                @{
+                    var IdFilterError = !Model.IsValid("IdFilter");
+                    var IdFilterGroupState = IdFilterError ? Error : Standard;
+                }
+                <nhs-form-group nhs-form-group-type=@IdFilterGroupState v-bind:class="{ 'nhsuk-form-group--error': hasError}">

```

Comment:
`hasError` here would require this to be run under a vue component, with a suitably declared data element - which as far as I can tell this isn't.

If this needs to be handled client site see ValidateInput.ts for an example of the adopted pattern that we're using moving forwards.
If we can get away with server-side errors only see ClinicalDetails.cshtml.

Reply:
Good spot - I was planning on adding dynamic validation but ended up not needing to and forgot to remove this, thanks :)